### DISCUSSION TITLE
Transparent background android viewer

### DISCUSSION THREAD

**Discussion by AdriFernan12** [google/model-viewer#3671] [Q&A] [answered]
Hi! I am trying to render a 3d object on a model-viewer on Android Studio with Kotlin, it works fine but it shows a white background that I want to remove. Do you know how can i do it? Thanks in advance!

**The code of the modelViewer.html:**

![image](https://user-images.githubusercontent.com/58808908/182820389-6db5fbc2-026b-4fa1-af36-1a5916265f36.png)

**Code of loadModel function:** 

![image](https://user-images.githubusercontent.com/58808908/182820270-6c3c4c80-2665-4f4a-97b4-3c373fb6eba6.png)

Result
![qPRUO](https://user-images.githubusercontent.com/58808908/182820124-4475b29a-4013-4d12-94ee-4a964dec5edb.png)

**Comment by elalish** [comment]
MV itself does render transparent; I'd guess your webview is putting in the default white background that most browsers do. I don't know much about Android development, but hopefully the webview has some configuration for that background, or for enabling alpha.

**Reply by AdriFernan12** [reply]
If i enable alpha it goes black and does not show nothing, webviewer on android has an option to put it on transparent but it continues showing me the white background.

**Comment by AdriFernan12** [comment, marked as answer]
Solved, thanks @elalish , i didn't know that MV does render transparent itself so i started taking a look to webviewer and not to ModelViewer, i had to set transparent color putting on my LoginActivity.kt this line:
webview.setBackgroundColor(Color.TRANSPARENT)
